browser/resources/settings/brave_overrides/clear_browsing_data_dialog.js

```diff
@@ -3,11 +3,32 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
+import {html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
 import {BraveClearBrowsingDataOnExitBehavior} from '../brave_clear_browsing_data_dialog/brave_clear_browsing_data_dialog_behavior.js'
-import {RegisterPolymerComponentBehaviors} from 'chrome://brave-resources/polymer_overriding.js'
+import {RegisterPolymerComponentBehaviors, RegisterPolymerTemplateModifications, RegisterStyleOverride} from 'chrome://brave-resources/polymer_overriding.js'
+import {loadTimeData} from '../i18n_setup.js'
+
+RegisterStyleOverride(
+  'settings-clear-browsing-data-dialog',
+  html`
+    <style>
+      #rewards-reset-data {
+        display: block;
+        margin-top: 10px;
+      }
+    </style>
+  `
+)
 
 RegisterPolymerComponentBehaviors({
   'settings-clear-browsing-data-dialog': [
     BraveClearBrowsingDataOnExitBehavior
   ]
 })
+
+RegisterPolymerTemplateModifications({
+  'settings-clear-browsing-data-dialog': (templateContent) => {
+    const body = templateContent.querySelector('[slot="body"]')
+    body.innerHTML = body.innerHTML + `<a id="rewards-reset-data" href="chrome://rewards/#manage-wallet">${loadTimeData.getString('resetRewardsData')}</a>`
```

Comment:
Sorry to be a pain, but changing `innerHTML` is less performant than `insertAdjacentHTML` (or `appendChild`), as it causes rebuilding the whole DOM. So if you're able, please change to
```js
body.insertAdjacentHTML('beforeend', `<a id="rewards-reset-data" href="chrome://rewards/#manage-wallet">${loadTimeData.getString('resetRewardsData')}</a>`)
```

Please also add a check to see if `body` was found, and otherwise log an error, so that we catch any problems on chromium rebase.
```js
if (!body) {
  console.error(`[Brave Settings Overrides] cannot find 'slot="body"' in clear-browsing-data-dialog`)
} else {
  ...
}
```

Reply:
sure not a problem. I just copied `innerHTML` from `about_page.js` code. Fixed